src/components/Toolbar.jsx

```diff
@@ -4,9 +4,9 @@ import React from 'react'
 import { translate } from '../plugins/preact-polyglot'
 
 const Toolbar = ({ t }) => (
-  <div class={styles['fil-content-toolbar']} role='toolbar'>
-    <button role='button' class='coz-btn coz-btn--regular coz-btn--upload'>{ t('toolbar.item_upload') }</button>
-    <button role='button' class='coz-btn coz-btn--more'><span class='coz-hidden'>{ t('toolbar.item_more') }</span></button>
+  <div className={styles['fil-content-toolbar']} role='toolbar'>
+    <button role='button' className='coz-btn coz-btn--regular coz-btn--upload'>{ t('toolbar.item_upload') }</button>
+    <button role='button' className='coz-btn coz-btn--more'><span class='coz-hidden'>{ t('toolbar.item_more') }</span></button>
```

Comment:
Do we have to use className or class attribute here? I thought pReact used class but maybe it's className using preact-compat. But I think we should not keep a mix of the two ways here. What do you think?